scripts/release_helper/python.py

```diff
@@ -1,23 +1,68 @@
-from common import IssueProcess, Common
-from typing import Any, List
 import os
+import re
+from typing import Any, List
+
+from common import IssueProcess, Common, get_origin_link_and_tag
+from utils import AUTO_CLOSE_LABEL, get_last_released_date, record_release
 
 # assignee dict which will be assigned to handle issues
-_PYTHON_OWNER = {'BigCat20196', 'msyyc'}
+_PYTHON_OWNER = {'BigCat20196', 'msyyc', 'azure-sdk'}
 
 # 'github assignee': 'token'
 _ASSIGNEE_TOKEN_PYTHON = {
     'BigCat20196': os.getenv('PYTHON_BIGCAT_TOKEN'),
     'msyyc': os.getenv('PYTHON_MSYYC_TOKEN'),
 }
```

Comment:
change to use `azure-sdk` token. Thus, there is no need to use dict and set shall be enough.